### ISSUE TITLE
fresh install, unable to disable developpement-mode

### ISSUE THREAD

**Issue by mickaelmonsieur** [zfcampus/zf-apigility-skeleton#155]
Hi,

I installed the current version of apigility and can not disable the development mode:

`
web2@vps104:/var/www/api.xx.net/web$ curl -sS https://apigility.org/install | php
Download Apigility
.....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
Install Apigility
Installation complete.
Running PHP internal web server.
Open your browser to http://localhost:8888, Ctrl-C to stop it.
^C
web2@vps104:/var/www/api.xx.net/web# ./vendor/bin/zf-development-mode disable
bash: ./vendor/bin/zf-development-mode: Permission non accordée (EN: Forbidden)
web2@vps104:/var/www/api.xx.net/web# chmod +x ./vendor/bin/zf-development-mode
web2@vps104:/var/www/api.xx.net/web$ ./vendor/bin/zf-development-mode disable
Cannot locate autoloader; please run "composer install"
web2@vps104:/var/www/api.xx.net/web$ composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
web2@vps104:/var/www/api.xx.net/web$ ./vendor/bin/zf-development-mode disable
Cannot locate autoloader; please run "composer install"
web2@vps104:/var/www/api.xx.net/web$ composer -v
   ______
  / ____/___  ____ ___  ____  ____  ________  _____
 / /   / __ \/ __ `__ \/ __ \/ __ \/ ___/ _ \/ ___/
/ /___/ /_/ / / / / / / /_/ / /_/ (__  )  __/ /
\____/\____/_/ /_/ /_/ .___/\____/____/\___/_/
                    /_/
Composer version 1.3.2 2017-01-27 18:23:41
`

[I dont' like COMPOSER, more difficult! :D]

**Comment by webimpress** [comment]
@mickaelmonsieur If you need solution for now I can suggest just to remove whole vendor directory and run `composer update` (or `composer install`) again. Then development mode should work as expected.

I've investigated it, and for me it seems to be wrong zip package with apigility-skeleton 1.4.1. We will try to resolve it somehow :) Thanks for reporting the issue!

**Comment by mickaelmonsieur** [comment]
Hi @webimpress,
Thanks for the solution!

I confirm:

`
web2@vps104:/var/www/api.xx.net/web$ mv vendor/ vendor_old
web2@vps104:/var/www/api.xx.net/web$ composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 61 installs, 0 updates, 0 removals
  - Installing zfcampus/zf-asset-manager (1.1.1) Downloading: 100%         
  - Installing zendframework/zend-component-installer (dev-master 28838e3) Cloning 28838e3683
  - Installing zendframework/zend-stdlib (3.1.0) Downloading: 100%         
  - Installing zendframework/zend-loader (2.5.1) Downloading: 100%         
  - Installing zendframework/zend-eventmanager (3.1.0) Downloading: 100%         
  - Installing zendframework/zend-view (2.8.1) Downloading: 100%         
  - Installing psr/container (1.0.0) Downloading: 100%         
  - Installing container-interop/container-interop (1.2.0) Downloading: 100%         
  - Installing zendframework/zend-servicemanager (3.2.1) Downloading: 100%         
  - Installing zendframework/zend-validator (2.8.2) Downloading: 100%         
  - Installing zendframework/zend-escaper (2.5.2) Downloading: 100%         
  - Installing zendframework/zend-uri (2.5.2) Downloading: 100%         
  - Installing zendframework/zend-http (2.6.0) Downloading: 100%         
  - Installing zendframework/zend-router (3.0.2) Downloading: 100%         
  - Installing zendframework/zend-config (2.6.0) Downloading: 100%         
  - Installing zendframework/zend-modulemanager (2.7.2) Downloading: 100%         
  - Installing zendframework/zend-mvc (3.0.4) Downloading: 100%         
  - Installing zendframework/zend-hydrator (2.2.1) Downloading: 100%         
  - Installing zendframework/zend-filter (2.7.1) Downloading: 100%         
  - Installing zendframework/zend-inputfilter (2.7.3) Downloading: 100%         
  - Installing zendframework/zend-code (3.1.0) Downloading: 100%         
  - Installing zendframework/zend-json (3.0.0) Downloading: 100%         
  - Installing zfcampus/zf-api-problem (1.2.2) Downloading: 100%         
  - Installing zfcampus/zf-content-negotiation (1.3.0) Downloading: 100%         
  - Installing zfcampus/zf-configuration (1.2.1) Downloading: 100%         
  - Installing zfcampus/zf-apigility-provider (1.2.0) Downloading: 100%         
  - Installing michelf/php-markdown (1.7.0) Downloading: 100%         
  - Installing zfcampus/zf-apigility-documentation (1.2.3) Downloading: 100%         
  - Installing zendframework/zend-paginator (2.7.0) Downloading: 100%         
  - Installing zendframework/zend-db (2.8.2) Downloading: 100%         
  - Installing paragonie/random_compat (v2.0.4) Downloading: 100%         
  - Installing ramsey/uuid (2.9.0) Downloading: 100%         
  - Installing zfcampus/statuslib-example (1.1.0) Downloading: 100%         
  - Installing zfcampus/zf-versioning (1.2.0) Downloading: 100%         
  - Installing zfcampus/zf-rpc (1.3.1) Downloading: 100%         
  - Installing zendframework/zend-math (3.0.0) Downloading: 100%         
  - Installing zendframework/zend-crypt (3.2.0) Downloading: 100%         
  - Installing bshaffer/oauth2-server-php (v1.9.0) Downloading: 100%         
  - Installing zfcampus/zf-oauth2 (1.4.0) Downloading: 100%         
  - Installing zendframework/zend-permissions-rbac (2.5.1) Downloading: 100%         
  - Installing zendframework/zend-permissions-acl (2.6.0) Downloading: 100%         
  - Installing zendframework/zend-authentication (2.5.3) Downloading: 100%         
  - Installing zfcampus/zf-mvc-auth (1.4.3) Downloading: 100%         
  - Installing zfcampus/zf-hal (1.4.2) Downloading: 100%         
  - Installing zfcampus/zf-rest (1.3.3) Downloading: 100%         
  - Installing zfcampus/zf-content-validation (1.3.5) Downloading: 100%         
  - Installing zfcampus/zf-apigility-admin-ui (1.3.9) Downloading: 100%         
  - Installing zfcampus/zf-apigility (1.3.0) Downloading: 100%         
  - Installing zfcampus/zf-apigility-admin (1.5.9) Downloading: 100%         
  - Installing zfcampus/zf-composer-autoloading (1.0.0) Downloading: 100%         
  - Installing zendframework/zend-console (2.6.0) Downloading: 100%         
  - Installing zfcampus/zf-console (1.3.0) Downloading: 100%         
  - Installing kherge/version (1.0.1) Downloading: 100%         
  - Installing seld/jsonlint (1.5.0) Downloading: 100%         
  - Installing justinrainbow/json-schema (1.6.1) Downloading: 100%         
  - Installing herrera-io/json (1.0.3) Downloading: 100%         
  - Installing herrera-io/phar-update (1.0.3) Downloading: 100%         
  - Installing zfcampus/zf-deploy (1.2.0) Downloading: 100%         
  - Installing zfcampus/zf-development-mode (3.1.0) Downloading: 100%         
  - Installing zendframework/zend-debug (2.5.1) Downloading: 100%         
  - Installing zendframework/zend-developer-tools (1.1.1) Downloading: 100%         
zendframework/zend-view suggests installing zendframework/zend-feed (Zend\Feed component)
zendframework/zend-view suggests installing zendframework/zend-i18n (Zend\I18n component)
zendframework/zend-view suggests installing zendframework/zend-navigation (Zend\Navigation component)
zendframework/zend-servicemanager suggests installing ocramius/proxy-manager (ProxyManager 1.* to handle lazy initialization of services)
zendframework/zend-validator suggests installing zendframework/zend-i18n (Zend\I18n component to allow translation of validation error messages as well as to use the various Date validators)
zendframework/zend-validator suggests installing zendframework/zend-i18n-resources (Translations of validator messages)
zendframework/zend-validator suggests installing zendframework/zend-session (Zend\Session component)
zendframework/zend-router suggests installing zendframework/zend-i18n (^2.6, if defining translatable HTTP path segments)
zendframework/zend-config suggests installing zendframework/zend-i18n (Zend\I18n component)
zendframework/zend-mvc suggests installing zendframework/zend-mvc-console (zend-mvc-console provides the ability to expose zend-mvc as a console application)
zendframework/zend-mvc suggests installing zendframework/zend-mvc-i18n (zend-mvc-i18n provides integration with zend-i18n, including a translation bridge and translatable route segments)
zendframework/zend-mvc suggests installing zendframework/zend-mvc-plugin-fileprg (To provide Post/Redirect/Get functionality around forms that container file uploads)
zendframework/zend-mvc suggests installing zendframework/zend-mvc-plugin-flashmessenger (To provide flash messaging capabilities between requests)
zendframework/zend-mvc suggests installing zendframework/zend-mvc-plugin-identity (To access the authenticated identity (per zend-authentication) in controllers)
zendframework/zend-mvc suggests installing zendframework/zend-mvc-plugin-prg (To provide Post/Redirect/Get functionality within controllers)
zendframework/zend-mvc suggests installing zendframework/zend-psr7bridge ((^0.2) To consume PSR-7 middleware within the MVC workflow)
zendframework/zend-mvc suggests installing zendframework/zend-servicemanager-di (zend-servicemanager-di provides utilities for integrating zend-di and zend-servicemanager in your zend-mvc application)
zendframework/zend-hydrator suggests installing zendframework/zend-serializer (^2.6.1, to use the SerializableStrategy)
zendframework/zend-filter suggests installing zendframework/zend-i18n (Zend\I18n component for filters depending on i18n functionality)
zendframework/zend-code suggests installing doctrine/annotations (Doctrine\Common\Annotations >=1.0 for annotation features)
zendframework/zend-json suggests installing zendframework/zend-json-server (For implementing JSON-RPC servers)
zendframework/zend-json suggests installing zendframework/zend-xml2json (For converting XML documents to JSON)
zendframework/zend-paginator suggests installing zendframework/zend-cache (Zend\Cache component to support cache features)
paragonie/random_compat suggests installing ext-libsodium (Provides a modern crypto API that can be used to generate random bytes.)
ramsey/uuid suggests installing moontoast/math (Support for converting UUID to 128-bit integer (in string form).)
ramsey/uuid suggests installing symfony/console (Support for use of the bin/uuid command line tool.)
ramsey/uuid suggests installing doctrine/dbal (Allow the use of a UUID as doctrine field type.)
zendframework/zend-math suggests installing ext-gmp (If using the gmp functionality)
bshaffer/oauth2-server-php suggests installing predis/predis (Required to use Redis storage)
bshaffer/oauth2-server-php suggests installing thobbs/phpcassa (Required to use Cassandra storage)
bshaffer/oauth2-server-php suggests installing aws/aws-sdk-php (~2.8 is required to use DynamoDB storage)
bshaffer/oauth2-server-php suggests installing firebase/php-jwt (~1.1 is required to use MondoDB storage)
zfcampus/zf-oauth2 suggests installing alcaeus/mongo-php-adapter (^1.0.5, if you are using ext/mongodb and wish to use the MongoAdapter for OAuth2 credential storage.)
zendframework/zend-authentication suggests installing zendframework/zend-ldap (Zend\Ldap component)
zendframework/zend-authentication suggests installing zendframework/zend-session (Zend\Session component)
zfcampus/zf-apigility-admin-ui suggests installing rwoverdijk/assetmanager (^1.7, to expose assets under the document root (may not yet be released))
zfcampus/zf-apigility suggests installing alcaeus/mongo-php-adapter (^1.0.5, if you are using ext/mongodb and wish to use the MongoConnectedListener.)
zfcampus/zf-apigility suggests installing rwoverdijk/assetmanager (^1.7, to install the assets used by zf-apigility-welcome and/or zf-apigility-admin-ui (may not yet be released))
zendframework/zend-debug suggests installing ext/xdebug (XDebug, for better backtrace output)
zendframework/zend-developer-tools suggests installing bjyoungblood/bjy-profiler (Version: dev-master, allows the usage of the (Zend) Db collector.)
zendframework/zend-developer-tools suggests installing ocramius/ocra-service-manager (OcraServiceManager can help you track dependencies within your application.)
zendframework/zend-developer-tools suggests installing san/san-session-toolbar (SanSessionToolbar can help you see current Zend\Session data you're using within your application.)
zendframework/zend-developer-tools suggests installing snapshotpl/zf-snap-event-debugger (ZfSnapEventDebugger can help you debug events from Zend\EventManager)
zendframework/zend-developer-tools suggests installing doctrine/doctrine-orm-module (Profile DoctrineORM queries)
zendframework/zend-developer-tools suggests installing jhuet/zdt-logger-module (Show you log data from Zend\Log)
zendframework/zend-developer-tools suggests installing aist/aist-git-tools (Show you informations about current GIT repository)
Package kherge/version is abandoned, you should avoid using it. No replacement was suggested.
Package herrera-io/json is abandoned, you should avoid using it. Use kherge/json instead.
Package herrera-io/phar-update is abandoned, you should avoid using it. No replacement was suggested.
Writing lock file
Generating autoload files
web2@vps104:/var/www/api.xx.net/web$ ./vendor/bin/zf-development-mode disable
Development mode is now disabled.
web2@vps104:/var/www/api.xx.net/web$ `

**Comment by mickaelmonsieur** [comment]
Off topic: I also think that github has a problem with the code <> tag ...

**Comment by weierophinney** [comment]
This repository has been closed and moved to laminas-api-tools/api-tools-skeleton; a new issue has been opened at https://github.com/laminas-api-tools/api-tools-skeleton/issues/7.
